src/cascadia/TerminalConnection/AzureConnection.cpp

```diff
@@ -79,6 +110,35 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
         _transitionToState(ConnectionState::Connecting);
     }
 
+    std::optional<std::wstring> AzureConnection::_ReadUserInput(InputMode mode)
+    {
+        std::unique_lock<std::mutex> inputLock{ _inputMutex };
+
+        if (_isStateAtOrBeyond(ConnectionState::Closing))
+        {
+            return std::nullopt;
+        }
+
+        _currentInputMode = mode;
+
+        _TerminalOutputHandlers(L"\x1b[92m"); // Make prompted user input green

```

Comment:
It's either this or we allocate a dynamically generated string just to insert the 92 into it because it needs to be string-formatted.

I guess this whole string could be a const.